src/fluent_linter/linter.py

```diff
@@ -272,6 +271,9 @@ def visit_Term(self, node):
         if "SY01" in self.config and self.config["SY01"]["disabled"]:
             self.add_error(node, node.id, "SY01", "Terms are not supported.")
 
+        # Validate the term identifier - terms should have valid identifiers after the hyphen
+        self._validate_id03(node, node.id.name, f"-{node.id.name}", True)
```

Comment:
I don't think you can hit this code, because you would be looking at junk and not a term.